app/scripts/components/reconciliation-reports/reconciliation-report.js

```diff
@@ -75,35 +68,119 @@ class ReconciliationReport extends React.Component {
     this.props.router.push('/reconciliations');
   }
 
+  getFilesSummary ({
+    onlyInDynamoDb = [],
+    onlyInS3 = []
+  }, {
+    onlyInCumulus = [],
+    onlyInCmr = []
+  }) {
+    const parseFileObject = (d) => {
+      const parsed = url.parse(d.uri);
+      return {
+        granuleId: d.granuleId,
+        filename: path.basename(parsed.pathname),
+        bucket: parsed.hostname,
+        path: parsed.href
+      };
+    };
+
+    const filesInS3 = onlyInS3.map(d => {
+      const parsed = url.parse(d);
+      return {
+        filename: path.basename(parsed.pathname),
+        bucket: parsed.hostname,
+        path: parsed.href
+      };
+    });
+
+    const filesInDynamoDb = onlyInDynamoDb.map(parseFileObject);
+
+    const filesOnlyInCumulus = onlyInCumulus.map(parseFileObject);
+
+    const filesOnlyInCmr = onlyInCmr.map(d => {
+      const parsed = url.parse(d.URL);
+      const bucket = parsed.hostname.split('.')[0];
+      return {
+        granuleId: d.GranuleUR,
+        filename: path.basename(parsed.pathname),
+        bucket,
+        path: `s3://${bucket}${parsed.pathname}`
+      };
+    });
+
+    return { filesInS3, filesInDynamoDb, filesOnlyInCumulus, filesOnlyInCmr };
```

Comment:
I don't know if it's a good idea to put the two `files` report together.
`filesInS3, filesInDynamoDb` is the result of comparing all the files in s3 and dynamodb.
`filesOnlyInCumulus, filesOnlyInCmr` is the result of comparing files for the granules which are in both Cumulus and Cmr.  So `filesOnlyInCumulus` doesn't include the granule files whose granule is not in CMR.  Put this two reports together may cause confusion.
I suggest to list the `filesOnlyInCumulus, filesOnlyInCmr` after the granule report.